habitat_baselines/rl/ppo/policy.py

```diff
@@ -156,11 +147,10 @@ def is_blind(self):
     def num_recurrent_layers(self):
         return self.state_encoder.num_recurrent_layers
 
-    def get_target_encoding(self, observations):
-        return observations[self.goal_sensor_uuid]
-
     def forward(self, observations, rnn_hidden_states, prev_actions, masks):
-        target_encoding = self.get_target_encoding(observations)
+        target_encoding = observations[
+            IntegratedPointGoalGPSAndCompassSensor.cls_uuid
```

Comment:
For both the SimpleCNN and ResNet agents, this needs look for both the `IntegratedPointGoalGPSAndCompassSensor` and the `PointGoalSensor` to keep PointNav w/o GPS+Compass working.

Reply:
Good catch! Surprised that isn't covered by our current unit tests.

Reply:
We don't have config with `PointNav` sensor only.